test/extended/cli/mustgather.go

```diff
@@ -227,6 +227,12 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 						}
 						o.Expect(text).To(o.HavePrefix(`{"kind":"Event",`))
 						for _, token := range []string{"oauthaccesstokens", "oauthauthorizetokens", tokenName} {
+							// We can safely ignore 404 coming from apiserver reaching oauth.
+							// This is because oauth-apiserver is separate entity and this event is coming
+							// from openshift-apiserver trying to reach it.
+							if strings.Contains(text, `"username":"system:apiserver"`) && strings.Contains(text, `"code":404`) {
```

Comment:
wait, when there is a 404, we get the token displayed?

Reply:
Sample event where we fail:
```
{"kind":"Event","apiVersion":"audit.k8s.io/v1","level":"Metadata","auditID":"46af2b9c-afca-4c93-ab20-6b7c052db13c","stage":"ResponseComplete","requestURI":"/apis/oauth.openshift.io/v1/oauthaccesstokens/TEST","verb":"get","user":{"username":"system:apiserver","groups":["system:masters","system:authenticated"]},"sourceIPs":["::1","10.128.0.1"],"userAgent":"kube-apiserver/v1.19.0+e465e66 (linux/amd64) kubernetes/e465e66","objectRef":{"resource":"oauthaccesstokens","name":"TEST","apiGroup":"oauth.openshift.io","apiVersion":"v1"},"responseStatus":{"metadata":{},"status":"Failure","reason":"NotFound","code":404},"requestReceivedTimestamp":"2020-09-26T21:00:34.017891Z","stageTimestamp":"2020-09-26T21:00:34.020104Z","annotations":{"authorization.k8s.io/decision":"allow","authorization.k8s.io/reason":""}}
```
I spoke with @sttts and we're going to skip this test on fresh 4.6+ clusters, for older we'll keep it.